maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -99,7 +103,6 @@ private void export(Run run) throws Exception {
 	private void resolve(StandaloneRun run) throws Exception, IOException {
 		try (ProjectResolver projectResolver = new ProjectResolver(run)) {
 			List<Container> runBundles = projectResolver.getRunBundles();
```

Comment:
This is done differently that the gradle plugin's resolve task: https://github.com/bndtools/bnd/blob/master/biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BndPlugin.groovy#L480-L491. We should use identical logic to avoid discrepancies.

I don't know which one is better, but they should be the same logic.